### ISSUE TITLE
Issue with `SlugField` (no-member, unsubscriptable-object)

### ISSUE THREAD

**Issue by sergafts** [landscapeio/pylint-django#89]
I have a `SlugField` in one of my models and I am trying to do string manipulation on it as such

```
self.some_field.split(' ')
self.some_field[:3]
```

The definition for `field` is 

```
some_field = models.SlugField(max_length=255)
```

I am getting:

```
[unsubscriptable-object] Value `self.some_field` is unsubscriptable
[no-member] Instance of `SlugField` has no `split` member
```

I thought that since `SlugField` inherits from `models.CharField` this is probably a false positive

**Comment by sergafts** [comment]
Probably misconfiguration on my end. Closing.
